app/mailers/report_mailer.rb

```diff
@@ -11,6 +11,11 @@ def deleted_user_accounts_report(email:, name:, issuers:, data:)
     mail(to: email, subject: t('report_mailer.deleted_accounts_report.subject'))
   end
 
+  def monthly_key_metrics_report(email:, data:)
```

Comment:
let's add a preview version of this to `report_mailer_preview.rb` and then we can add screenshots of it

Reply:
Zach took care of this himself in this PR: https://github.com/18F/identity-idp/pull/9219